common/djangoapps/student/models.py

```diff
@@ -1607,6 +1621,25 @@ def enroll(cls, user, course_key, mode=None, check_access=False, can_upgrade=Fal
         enrollment.update_enrollment(is_active=True, mode=mode, enterprise_uuid=enterprise_uuid)
         enrollment.send_signal(EnrollStatusChange.enroll)
 
+        # Announce user's enrollment
+        COURSE_ENROLLMENT_CREATED.send_event(
+            enrollment=CourseEnrollmentData(
+                user=UserData(
+                    pii=UserPersonalData(
```

Comment:
Love the abstraction layers that are being created for these data objects as part of this effort!